module/behaviour/Controller/src/Controller.cpp

```diff
@@ -37,16 +37,15 @@ namespace module::behaviour {
     using iterators = std::pair<std::vector<std::reference_wrapper<RequestItem>>::iterator,
                                 std::vector<std::reference_wrapper<RequestItem>>::iterator>;
 
-    Controller::Controller(std::unique_ptr<NUClear::Environment> environment)
-        : Reactor(std::move(environment)), actions(), limbAccess(), requests(), currentActions(), commandQueues() {
```

Comment:
I'm assuming that these still get default initialised.

Reply:
yesszir